bio_hansel/qc/checks.py

```diff
@@ -41,16 +41,56 @@ def is_missing_tiles(st: Subtype, df: DataFrame, p: SubtypingParams) -> Tuple[Op
     Returns:
         None, None if less missing tiles than tolerate; otherwise, "FAIL", error message
     """
-    if not st.are_subtypes_consistent:
-        return None, None
+    error_status = None
+    error_messages = None
+
+    if st.are_subtypes_consistent:
+        error_status, error_messages = check_for_missing_tiles(is_fastq=st.is_fastq_input(),
+                                                               curr_subtype=st.subtype,
+                                                               df=df,
+                                                               exp=int(st.n_tiles_matching_all_expected),
+                                                               obs=int(st.n_tiles_matching_all),
+                                                               p=p)
+    else:
+        error_status = []
+        error_messages = []
+
+        subtype_list = st.subtype.split(';')
+        n_tiles_matching_all_expected = st.n_tiles_matching_all_expected.split(';')
+        n_tiles_matching_subtype_exp = st.n_tiles_matching_positive_expected.split(';')
+
+        dfpos = df[df.is_pos_tile]
+        mixed_subtype_counts = get_mixed_subtype_tile_counts(dfpos=dfpos,
+                                                             subtype_list=subtype_list)
+        for curr_subtype, exp, st_exp in zip(subtype_list, n_tiles_matching_all_expected, n_tiles_matching_subtype_exp):
+
+            curr_status, curr_messages = check_for_missing_tiles(is_fastq=st.is_fastq_input(),
+                                                                 curr_subtype=curr_subtype,
+                                                                 df=df,
+                                                                 exp=(int(exp) + int(st_exp)),
+                                                                 obs=(mixed_subtype_counts.get(curr_subtype) +
```

Comment:
Wouldn't this need to be the number of matching tiles for the subtype + all matching negative only tiles?

Reply:
Ah this is what I was wondering as referenced to the comment above.